apps/admin-page/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -1,10 +1,397 @@
-import { Component } from '@angular/core';
+/* eslint-disable prefer-const */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
```

Comment:
There is no code present that would require those, and even if it did exist ti would be better to fix them.
Please remove these eslint rule ignore comments
```suggestion
```